doozer/doozerlib/cli/scan_sources_konflux.py

```diff
@@ -417,18 +417,10 @@ async def scan_network_mode_changes(self, image_meta: ImageMetadata):
         self.logger.debug(f"Network mode of {image_meta.name} in config is {network_mode}")
         build_record = self.latest_image_build_records_map[image_meta.distgit_key]
 
-        # Inspect the SLSA provenance: https://konflux.pages.redhat.com/docs/users/metadata/attestations.html
-        # To see if the latest build is actually hermetic
-        cmd = f"cosign download attestation {build_record.image_pullspec} --registry-username {os.environ['KONFLUX_ART_IMAGES_USERNAME']} --registry-password {os.environ['KONFLUX_ART_IMAGES_PASSWORD']}"
-        rc, attestation, error = await exectools.cmd_gather_async(cmd)
-        if rc != 0:
-            raise IOError(
-                f"Failed to get SLSA attestation for {build_record.image_pullspec}: {error}"
-            )
-
+        attestation = await artcommonlib.util.get_konflux_slsa_attestation(build_record.image_pullspec)
         try:
             # Equivalent bash code: jq -r ' .payload | @base64d | fromjson | .predicate.invocation.parameters.hermetic'
-            payload_json = json.loads(base64.b64decode(json.loads(attestation.strip())["payload"]).decode("utf-8"))
+            payload_json = json.loads(base64.b64decode(json.loads(attestation)["payload"]).decode("utf-8"))
         except Exception as e:
             raise IOError(f"Failed to parse SLSA attestation for {build_record.image_pullspec}: {e}")
         is_hermetic = payload_json["predicate"]["invocation"]["parameters"]["hermetic"]
```

Comment:
if this is supposed to be a bool param, then maybe we should convert it to bool (instead of str for comparison later on) - and if it is not a bool, complain!

Reply:
its a bool param but its retrieved as a string, I've made an update in https://github.com/openshift-eng/art-tools/pull/1570/commits/e7f988d9c6288eea1bebb7772a03797f100cead3

```
is_hermetic = True if is_hermetic.lower() == "true" else False
```
if that works

Reply:
yup! that definitely reads better